File: Labs/lab03/randomnum.js
function random_namber(min, max=null) {
    let actual_min, actual_max;
    
    if (max === null){
        actual_min = 0;
        actual_max = min;
     } else {
        actual_min = min;
        actual_max = max;
     }
        
    const number = Math.floor(Math.random() * (actual_max - actual_min + 1)) + actual_min
    return number
}

console.log(random_namber(100))
console.log(random_namber(50, 100))

File: Labs/lab03/randomnum.py
import random 

def random_number(min, max=None):
    if max is None:
        max = min
        min = 0
    number = int(random.random() * (max - min + 1)) + min
    return number

print(random_number(0, 6))
print(random_number(5))
